eduNEXT/platform-global-teacher-campus
# feat: added tests for views

This PR adds basic testing for the happy path to submit a course for validation until it gets approved by a validator. It also adds a test to check the case where a validation body is exempt to skip the normal validation process.

**How to test**

1. Inside your platform-global-teacher-campus folder install a virtual environment
2. Run the command virtualenv -p python3.8 venv
3. Activate the environment by using source venv/bin/activate
4. Install the requirements by running pip install -r requirements/test.txt
5. Create an env variable for tox: export TOX_ENV=py38-django32
6.  Set the newly created varaible export TOXENV=${TOX_ENV//./}
7. With your activated environmment run tox
8. You should see the 3 tests passing.
